packages/mobile/src/page/Notice/Notification/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Link, useLocation } from "react-router-dom";
+
+import dayjs from "dayjs";
+
+import $ from "./style.module.scss";
+
+type Props = {
+  notification: {
+    id: number;
+    title: string;
+    dates: string;
+    hits: number;
+    breadcrumb: string;
+    scraps: number;
+  };
+};
+
+function Notification({ notification }: Props) {
+  const { id, breadcrumb, title, dates, hits, scraps } = notification;
+  const { pathname } = useLocation();
+  return (
+    <Link to={`${pathname}/${id}`} key={id}>
+      <div className={$.notification}>
+        <div className={$.breadcrumb}>{breadcrumb}</div>
+        <div className={$.title}>{title}</div>
```

Comment:
요런 것들 `span` 태그에 담는 건 어떨까요?
```suggestion
        <span className={$.title}>{title}</span>
```